docfx.json

```
@@ -16,6 +16,7 @@
         "dest": ".",
         "exclude": [
           "samples/**/*.*",
+          "**/snippets/**/*.*",
```

I think the final `*.*` is unnecessary:

```suggestion
          "**/snippets/**,
```